package.json

```diff
@@ -30,6 +30,8 @@
     "test-security": "npm run check_security && npm run audit_deps && node ./build/commands/scripts/commands.js start --enable_brave_update --network_log --user_data_dir_name=brave-network-test",
     "tslint": "tslint --project tsconfig-lint.json \"components/**/*.{ts,tsx}\"",
     "pep8": "pycodestyle --max-line-length 120 -r script",
+    "pylint": "pylint -j0 -ry script",
+    "pylint-report": "pylint -j0 -fparseable -rn --exit-zero script >pylint-report.txt",
```

Comment:
I followed the older `pep8` here and only covered the script folder, but maybe we should cover all of them - build, components, installer, script and tools?

Reply:
@bridiver what do you think?

Reply:
ideally we should cover all of them

Reply:
let's do that then

Reply:
done